style.css

```
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: url('./images/Group.png');
+  background-size: 180px;
+  background-position: right 1px bottom -230px;
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+}
+
+.empty {
+  margin: 60px;
+}
+
+.info {
+  /* margin-bottom: 20px; */
+  height: 40px;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  background-color: #3c3a39;
+}
+
+.my-info {
+  padding-top: 20px;
+}
+
+.my-info a {
+  list-style: none;
+  text-decoration: none;
+  color: #fff;
+}
+
+.my-info .name {
+  font-size: 26px;
+}
+
+.my-info .harmburger {
+  position: absolute;
+  right: 0;
+  margin-right: 40px;
+  top: 25px;
+}
+
+.section-container {
+  color: #fff;
+  margin-top: -20px;
+  padding: 10px;
+}
+
+.intro h1 {
+  color: #ff6b00;
+  font-size: 66px;
+  letter-spacing: 0.03em;
+  line-height: 64px;
+  padding-bottom: 10px;
+  padding-top: 40px;
+  margin-top: 60px;
+}
+
+.intro .about-me {
+  font-size: 26px;
+  line-height: 32px;
+}
+
+.about {
+  font-size: 40px;
+  padding: 8px 0;
+}
+
+.social-icon {
+  display: flex;
+  align-self: flex-start;
+}
```

- [ ] How about adding `list-style-type: none` to your `ul` tags so that they do not have bullet points on top of the icons like they currently do.
```suggestion
.social-icon {
  display: flex;
  align-self: flex-start;
  list-style-type: none;
}
```